pkg/streamingpromql/operators/aggregation.go

```diff
@@ -96,6 +97,7 @@ type group struct {
 
 	// Sum, presence, and histograms for each step.
 	floatSums           []float64
+	floatMeans          []float64 // Mean, or "compensating value" for Kahan summation.
```

Comment:
Would `floatCompensatingValues` or something like that be a clearer name?

Reply:
Perhaps. We may use this for mean's in the future, but will rename for now (can always go back)